temp/legacy-code/src/main/java/com/ivy/legacy/domain/action/settings/SettingsAct.kt

```diff
@@ -13,4 +14,14 @@ class SettingsAct @Inject constructor(
     override suspend fun Unit.compose(): suspend () -> Settings = suspend {
         io { settingsDao.findFirst() }
     } then { it.toDomain() }
-}
+
+    suspend fun getSettingsWithNextTheme(): Settings {
```

Comment:
I'm not sure if this is the best place for this function. I'm open to suggestions.

Reply:
It's fine, the legacy code is crappy